papis/commands/init.py

```diff
@@ -0,0 +1,87 @@
+r"""
+This command initializes a papis library interactively.
+
+Go to the directory where you want to have your papers
+
+::
+
+        mkdir -p ~/Documents/papers
+        cd ~/Documents/papers
+
+::
+
+        papis init
+
+and answer the questions.
+"""
+from typing import Optional  # noqa: F401
+import os
+
+import click
+
+import papis.utils
+import papis.config
+import papis.tui.utils
+import papis.logging
+
+logger = papis.logging.get_logger(__name__)
+
+INIT_PROMPTS = ["opentool", "editor", "use-git", "notes-name"]
+
+
+@click.command("init")
+@click.help_option("--help", "-h")
+@click.argument(
+    "dir_path",
+    required=False,
+    type=click.Path(),
+    default=None,
+    metavar="<LIBRARY DIRECTORY>",
+    nargs=1,
+)
+def cli(dir_path: Optional[str]) -> None:
+    """Initialize a papis library"""
+
+    config = papis.config.get_configuration()
+    defaults = papis.config.get_default_settings()
+    config_file = papis.config.get_config_file()
+
+    if os.path.exists(config_file):
+        logger.warning(
+            "Config file %s already exists,"
+            " this command may change some of its contents,", config_file)
+        if not papis.tui.utils.confirm("Do you want to continue?"):
+            return
```

Comment:
AFAIK comments might disappear, I think, and indentation might also be different when written, this might freak out some OCD folks like me, but I can test

Reply:
Hm, that's a good point. I didn't think about comments.. then yes, a nice big warning is likely a good idea!